release_notes/markdown.py

```diff
@@ -178,3 +180,56 @@ def render(notes: set[rnm.ReleaseNote]):
                     message = note.source_block.note_message
                     objs.append(list_item_from_note(message, note, group))
     return objs
+
+
+def component_resources(
```

Comment:
factor-out conversion of resource (actually it would be sufficient to accept _artefact_, considering we only care about type + access-type), like so:

```
def release_notes_for_ocm_component(component):
   "collect release note string for data gathered from ocm-component - including artefacts"


def release_notes_for_ocm_artefact(artefact):
  "check if type + access-type are supported, return None if not"
  "create release-notes-markdown-fragment for artefact and return it"
```